PROV/prov-cco-directmappings.ttl

```diff
@@ -19,46 +19,68 @@
-prov:wasAssociatedWith    rdfs:subPropertyOf cco:has_agent .
-prov:wasInvalidatedBy     rdfs:subPropertyOf cco:is_affected_by .
-prov:wasGeneratedBy       rdfs:subPropertyOf cco:is_output_of .
-prov:wasInformedBy        rdfs:subPropertyOf cco:caused_by .
-
-prov:invalidated          rdfs:subPropertyOf cco:affects .
-prov:used                 rdfs:subPropertyOf cco:affects .
-prov:generated            rdfs:subPropertyOf cco:has_output .
+prov:wasEndedBy           rdfs:subPropertyOf cco:affects ;
+                          rdfs:comment ""@en .
+prov:wasStartedBy         rdfs:subPropertyOf cco:affects ;
+                          rdfs:comment ""@en .
+prov:wasAssociatedWith    rdfs:subPropertyOf cco:has_agent ;
+                          rdfs:comment "An Activity that was associated with an Agent has the Agent as its agent."@en .
+prov:wasInvalidatedBy     rdfs:subPropertyOf cco:is_affected_by ;
+                          rdfs:comment "An Entity that was invalidated by an Activity is also affected by that Activity."@en .
+prov:wasGeneratedBy       rdfs:subPropertyOf cco:is_output_of ;
+                          rdfs:comment "An Entity that was generated by an Activity is output of that Activity"@en .
+prov:wasInformedBy        rdfs:subPropertyOf cco:caused_by ;
+                          rdfs:comment "If the Activity A1 was informed by the Activity A2, then A1 is caused by A2"@en .
+
+prov:invalidated          rdfs:subPropertyOf cco:affects ;
+                          rdfs:comment "An Activity that invalidated an Entity also affects that Entity."@en .
+prov:used                 rdfs:subPropertyOf cco:affects ;
+                          rdfs:comment "An Activity that used an Entity affects the Entity."@en .
+prov:generated            rdfs:subPropertyOf cco:has_output ;
+                          rdfs:comment "An Activity that generated an Entity has_output that Entity"@en .
 
 
 # "The _optional_ Plan adopted by an Agent in Association with some Activity. Plan specifications are out of the scope of this specification."
-prov:hadPlan                  rdfs:subPropertyOf cco:has_input . # an Influence has_input some Plan
+prov:hadPlan                  rdfs:subPropertyOf cco:has_input ; # an Influence has_input some Plan
+                              rdfs:comment "If an Association had plan some Plan, then it had that Plan as an input"@en .
 
 # "If this Activity prov:wasAssociatedWith Agent :ag, then it can qualify the Association using prov:qualifiedAssociation [ a prov:Association;  prov:agent :ag; :foo :bar ]."
-prov:qualifiedAssociation     rdfs:subPropertyOf  cco:has_process_part .
-prov:qualifiedAttribution     rdfs:subPropertyOf cco:is_affected_by .
-prov:qualifiedRevision        rdfs:subPropertyOf cco:is_affected_by .
+prov:qualifiedAssociation     rdfs:subPropertyOf  cco:has_process_part ;
+                              rdfs:comment "An Activity that is associated with an Agent has that Association has a process part"@en .
```

Comment:
It seems that "Agent" should not occur in this justification. Maybe it can be revised as: An activity that has an Association as its qualified association also has the Activity as its process part?

Reply:
Yes I agree we can remove that, I was trying to add more context since these are very confusing. Agent is referenced here because one comment on qualifiedAssociation is:
> If this Activity prov:wasAssociatedWith Agent :ag, then it can qualify the Association using prov:qualifiedAssociation [ a prov:Association;  prov:agent :ag; :foo :bar ].
But now we've removed Agent from the restrictions on Activity anyway, so I'll remove from this justification